Tests/test_SeqIO_UniprotIO.py

```diff
@@ -10,11 +10,9 @@
 
 from seq_tests_common import SeqRecordTestBaseClass
 
-from Bio import SeqIO
+from Bio import BiopythonDeprecationWarning, SeqIO
 from Bio.SeqRecord import SeqRecord
 
-from Bio import BiopythonDeprecationWarning
-
```

Comment:
You didn't need to change the imports (general pull request advice is to limit any style changes to the code being worked on).

Reply:
We should revisit #2828 - in the interim I have gotten used to automated import sorting (several different styles depending on who got to pick). Perhaps you'd have some fresh insight?